### ISSUE TITLE
hapiJwt2KeyAsync is undefined

### ISSUE THREAD

**Issue by afgallo** [auth0/node-jwks-rsa#42]
When using version **1.2.1**, the function `hapiJwt2KeyAsync` is undefined. See below:

```
        options: {
            complete: true,
            key: JwksRsa.hapiJwt2Key({ // if changed to hapiJwt2KeyAsync this would fail
                cache: true,
                rateLimit: true,
                jwksRequestsPerMinute: 5,
                jwksUri: options.auth0Uri
            }),
            verifyOptions: {
                audience: options.auth0Audience,
                issuer: options.auth0Issuer,
                algorithms: ['RS256']
            },
            validate: internals.validate
        }
```

Any thoughts?

Thanks!

**Comment by GUL-** [comment]
`hapiJwt2KeyAsync` is not defined yet, I had to fork and add it so using it with Api v17 is working.
See [https://github.com/GUL-/node-jwks-rsa](https://github.com/GUL-/node-jwks-rsa)

**Comment by afgallo** [comment]
Thanks @GUL- . Has this been releases yet?

**Comment by afgallo** [comment]
I can confirm this has been fixed in version #44. Thanks everyone!
